README.md

```diff
@@ -1,67 +1,232 @@
-Very basic README for now...
+# Forgerock Do (frodo)
 
 This is the ForgeROck DO (frodo) CLI executable. This is a statically linked binary which can be cross compiled for multiple platforms (Linux, MacOS, Windows etc.).
 
-# Features
-## Current
+## Features
+
+### Current
+
 - Obtain ForgeRock session token and admin access_tokens for a ForgeRock Identity Cloud or platform (ForgeOps) deployment
 - Saving and reading credentials (for multiple ForgeRock deployments) from a configuration file.
 - Export, import and pruning of journeys. Export includes scripts referenced by scripted decision nodes and when used with Identity Cloud or ForgeOps, `frodo` also includes Email Templates referenced by the Email Template or Email Suspend nodes.
 - Export of IDM configuration
 - Configuring a ForgeRock cloud tenant for a specific use case, also called a "recipe"
 
-## Future
+### Future
+
 - Importing IDM configuration
 - Parameter substitution when exporting and importing IDM configuration
 - Export and import of other ForgeRock ID Cloud configurations, like OAuth clients, SAML entities etc.
 - Export and import of data in ForgeRock ID Cloud
 
 ## Limitations
+
 `frodo` can't export passwords (including API secrets, etc), so these need to be manually added back to an imported tree or alternatively, export the source tree to a file, edit the file to add the missing fields before importing. Any dependencies _other than_ scripts and email templates, needed for a journey/tree, must also exist prior to import, for example inner-trees and custom nodes.
 
-# Quick start
+## Usage
+
 Depending on your OS, you can invoke `frodo-linux`, `frodo-macos` or `frodo-windows.exe` binary.
```

Comment:
The new packaging allows us to have a "frodo" command for each platform, so no more frodo-linux, frodo-macos, and frodo-win.exe. It's now just frodo and frodo.exe.